src/lib.rs

```diff
@@ -110,11 +110,12 @@ impl<T> AtomiQueue<T> {
         Ok(Some(value))
     }
 
-    /// Clones the top value off the queue.
+    /// Clones the top value off the front of the queue.
```

Comment:
This should probably be phrased as `Clones the oldest value`

Reply:
Changed